web-components/src/components/chip/Chip.ts

```diff
@@ -35,6 +35,7 @@ export namespace Chip {
     @property({ type: String }) role: Chip.Role = "group";
     @property({ type: String, reflect: true }) id = "";
     @property({ type: Boolean }) small = false;
+    @property({ type: Boolean }) interactive = true;
     @property({ type: Boolean }) readonly = false;
```

Comment:
interactive is the default so i would add decorative as an attribute. That means it can be modified as an attribute not just a property